README.md

```diff
@@ -56,7 +56,7 @@ More info: https://docs.gitea.com/installation/install-from-source
     ./gitea web
 
 > [!NOTE]
-> If you're interested in using our APIs, we have experimental support with [documentation](https://try.gitea.io/api/swagger).
+> If you're interested in using our APIs, we have experimental support with [documentation](https://gitea.com/api/swagger).
```

Comment:
`https://demo.gitea.com/api/swagger` would better to see latest api?

```suggestion
> If you're interested in using our APIs, we have experimental support with [documentation](https://demo.gitea.com/api/swagger).
```

Reply:
I changed it to https://docs.gitea.com/api/